Gemfile.lock

```diff
@@ -443,6 +443,7 @@ GEM
     wmi-lite (1.0.7)
 
 PLATFORMS
+  arm64-darwin-21
```

Comment:
@blabade Can you remove the root `Gemfile.lock` changes (the platforms update and bundled with are the only updates and should be unnecessary)

Reply:
Sure @tpowell-progress !
Actually I have discussed with @johnmccrae  on this his suggestion is that keeping this will not cause any issue.
As suggested removing this now.